Xamarin.Forms.Platform.Android/ResourceManager.cs

```diff
@@ -89,9 +92,17 @@ public static Drawable GetDrawable(this Resources resource, string name)
 			return AndroidAppCompat.GetDrawable(Forms.Context, id);
 		}
 
+		public static void LogInfo(string message)
```

Comment:
I just left this in so we can easily reference it for logging previewer messages